### PR TITLE
export varibles replicas to values.yaml

### PR DESCRIPTION
**Pull request by onlymellb**
### What problem does this PR solve? 
fix undefined variables `replicas` 

### Check List 

Tests 

 - Manual test 

Code changes

 - Has Helm charts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:
 
 ```release-note

 ```

### CODE REVIEW

File: charts/tidb-cluster/values.yaml [hunk 1]
```diff
@@ -706,6 +706,7 @@ scheduledBackup:
 
 importer:
   create: false
+  replicas: 1
```
Comment (tennix) [inline charts/tidb-cluster/values.yaml]: IIRC, TiDB Lightning cannot use mulitple tikv-importer to import data right now.

Comment (onlymellb) [reply charts/tidb-cluster/values.yaml]: Okay，then we can use number 1 instead of `{{ .Values.importer.replicas }}`

Comment (onlymellb) [reply charts/tidb-cluster/values.yaml]: In fact, we can also use multiple sets of `tidb-lighting` and `tikv-importer` at the same time.

**Pull request merged**